crazyflie/scripts/crazyflie_server.py

```diff
@@ -767,6 +767,17 @@ def _init_parameters(self):
                                 value=cf_param_value,
                                 descriptor=parameter_descriptor,
                                 )
+                            # Based on the parameters from the last Crazyflie, set params for all
+                            # Warning: if any of the other crazyflies have different paramters    this will result in an error
```

Comment:
```suggestion
                            # Warning: if any of the other crazyflies have different parameters
                            #                this will result in an error
```